src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -21,8 +21,8 @@
 
 // clang-format off
 #if defined(KOKKOS_ENABLE_CUDA)
-#  if defined(KOKKOS_COMPILER_CLANG) && KOKKOS_COMPILER_CLANG < 900
-// Clang of version less than 9.0 cannot compile Thrust, failing with errors
+#  if defined(KOKKOS_COMPILER_CLANG)
+// A range of versions of Clang cannot compile Thrust, failing with errors
```

Comment:
Please specify at least which ones you tried.

Reply:
See description above.  Not planning to include that in the comment.

Reply:
I would prefer having a comment in the code, as I would not remember to look at github PR description. In general, I think github does not substitute the need for good description in commit messages nor code comments.